learning-team/2020-09/2020-09-24.md

```diff
@@ -0,0 +1,51 @@
+# September 24, 2020
+
+Facilitator & Note Taker: 
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+- [ ] Amy Lam (AL)
+- [ ] Chris Manson (CM)
+- [ ] Isaac Lee (IL)
+- [ ] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Jessica Jordan (JJ)
+- [ ] Kenneth Larsen (KL)
+- [ ] Melanie Sumner (MS)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+
+### Guests
+- [] Your name here (INITIALS)
+
+## Weekly review
+
+### Review action items (weekly)
+See: [Action items board](https://github.com/orgs/ember-learn/projects/47)
+
+### Review Initiatives board (every other week)
+See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)
+
+### Review PR queue
+See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
+
+## Topics
+
+<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
+<!-- ### Your topic (INITIALS, expected duration in minutes) -->
+<!-- replace with topic -->
+<!-- replace with topic -->
+### Follow-up to Ember Guides feedback (IL, 5-10 min)
+
+If there is a quorum, let's follow up on last week's discussion on [conveying better a change in code](https://github.com/emberjs/core-notes/blob/master/learning-team/2020-09/2020-09-17.md#feedback-for-ember-guides-il-10-min)? Additional feedback can be found in https://github.com/ember-learn/guides-source/issues/1534.
+<!-- replace with topic -->
+<!-- replace with topic -->
+
+## Ember Times suggestions
```

Comment:
```suggestion
### RFC awareness (RM, 15min)
There have been two RFCs merged recently that impact our work more directly, [Deprecation Staging](https://github.com/emberjs/rfcs/blob/master/text/0649-deprecation-staging.md) and [RFC Stages](https://github.com/emberjs/rfcs/blob/master/text/0617-rfc-stages.md).
I think it would be nice to take some minutes to make sure everyone in the team is levelled up.

## Ember Times suggestions
```